app/cards/Card.tsx

```diff
@@ -0,0 +1,68 @@
+import * as marked from 'marked';
+import * as React from 'react';
+import { gql, graphql } from 'react-apollo';
+import { Card as ICard } from '../../server/models/';
+
+interface Props {
+  data: {
+    card: ICard;
+    networkStatus: number;
+  };
+  match: {
+    params: {
+      id: string;
+    };
+  };
+}
+
+const cardDataFromId = gql`
+  query getCardById($id: String!) {
+    card(id: $id) {
+      id
+      title
+      authors {
+        name
+      }
+      created
+      updates
+      content
+    }
+  }
+`;
+
+const config = {
+  options: ({ match }) => ({ variables: { id: match.params.id } }),
```

Comment:
For the life of me, can't figure out where `match` variable gets set to be passed in to these config options